lang_interface/java/com/intel/daal/utils/LibUtils.java

```diff
@@ -80,19 +80,19 @@ private static void loadFromJar(String path, String name) throws Throwable
 
         File fileOut = createTempFile(path, FullName);
         if (fileOut == null) {
-            logger.log(logLevel, "DONE: Loading library " + FullName + " as resource.");
+            logger.log(logLevel, "DONE: Loading library as resource.");
             return;
         }
 
         InputStream streamIn = LibUtils.class.getResourceAsStream(LIBRARY_PATH_IN_JAR + "/" + FullName);
         if (streamIn == null)
         {
-            throw new IOException("Error: No resource " + LIBRARY_PATH_IN_JAR + "/" + FullName + " found.");
```

Comment:
LIBRARY_PATH_IN_JAR is a constant, which can be viewed in sources. I'm not sure that it is a security problem. But okay.